sdk/textanalytics/Azure.AI.TextAnalytics/tests/Infrastructure/TextAnalyticsTestEnvironment.cs

```diff
@@ -9,6 +9,11 @@ public class TextAnalyticsTestEnvironment: TestEnvironment
     {
         public string Endpoint => GetRecordedVariable("TEXT_ANALYTICS_ENDPOINT");
         public string ApiKey => GetRecordedVariable("TEXT_ANALYTICS_API_KEY", options => options.IsSecret());
+
+        // The following variables refer to the static test resource, which is
+        // used to test the custom text analysis features.
+        public string StaticEndpoint => GetRecordedVariable("TEXT_ANALYTICS_STATIC_ENDPOINT");
+        public string StaticApiKey => GetRecordedVariable("TEXT_ANALYTICS_STATIC_API_KEY", options => options.IsSecret());
```

Comment:
I commented below as well, but you might - for consistency and ease of look-up (see how we use the service directory e.g., "textanalytics", in `TestEnvironment`) - use "TEXTANALYTICS" as a consistent prefix.